### DISCUSSION TITLE
Hot Keys

### DISCUSSION THREAD

**Discussion by snehanie** [memcached/memcached#1129] [Q&A]
Once a while there is particular hot key or keys which causes spikes in NetworkInterface transmitBytes. Is there a command to find hot keys like lru_crawler and stats. I found that with lru_crawler and stats command there is no way to find hot keys (Correct if i am wrong here). I looking for ways to do with console commands rather than network traffic sniffing tools like mctop, memsniff.

**Comment by dormando** [comment]
Since we don't internally track "hot keys" you will have to use the logging interface and a script to read a snapshot of data. you can do this with: `watch fetchers` (or `watch mutations`). See `doc/protocol.txt` - note that writes cause a fetch (to see if the key pre-exists) and then a write, so fetchers should actually represent all keys.
